config/default.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
       remotePath: '',
       disableUpload: false,
     },
+    apiKey: '',
```

Comment:
Can we add this to `custom-environment-variables.js`? @giorgiosironi ... do you have an opinion?